nexus/db-queries/src/db/datastore/sled.rs

```diff
@@ -54,16 +57,62 @@ use uuid::Uuid;
 
 #[derive(Debug, thiserror::Error)]
 enum SledReservationError {
-    #[error("No reservation could be found")]
+    #[error(
+        "Could not find any valid sled on which this instance can be placed"
+    )]
     NotFound,
-    #[error("More than one sled was found with 'affinity = required'")]
+    #[error("This instance belongs to an affinity group that requires it be placed \
+             on more than one sled. Instances can only placed on a single sled, so \
+             this is impossible to satisfy - Consider stopping other instances in \
```

Comment:
turbo nit: `this is impossible to satisfy. Consider...` to match the punctuation on line 80; same thing on line 73

Reply:
Updated!